contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -558,7 +567,7 @@ exec_stmt_push_result(PLtsql_execstate *estate,
 					 receiver,
 					 receiver,
 					 &qc))
-                processed = portal->portalPos;
+				processed = portal->portalPos;
```

Comment:
indent?

Reply:
I think this was an auto indent change from tabs to spaces.  Its indented by 4 tabs, arguments to the if are 5 tabs.  I'm not sure what are convention is when there's this many tabs.  I can change this line back to spaces if that's preferable.